src/Alert/Alert.js

```diff
@@ -24,21 +24,27 @@ const Alert = ({ children, onDismiss, type }) => {
   return (
     <div className={alertClass}>
       {onDismiss &&
-        <button type='button' className='close' aria-hidden='true' onClick={onDismiss}>
-          <span className='pficon pficon-close' />
+        <button
+          type="button"
```

Comment:
This rule is one of the collisions between prettier and srandardjs. eslint-config-prettier is taking care of this.  You'll need to disable your standardjs plugin in your editor and use eslint based plugin. Technically we're not using standardjs any more, we're just using it's eslint config.

Personally I very much like what Prettier does - use double quotes in jsx...